src/opt_handlers.c

```diff
@@ -676,6 +678,14 @@ options[] = {
 	  OPT_STRLIST, 0, NULL, &trashdir_handler, NULL,
 	  { .init = &init_trashdir },
 	},
+	{ "trash_filefmt_zero", "", "file name when moved to the trash directory and no other files exist with the same name",
+	  OPT_STR, 0, NULL, &trash_filefmt_zero_handler, NULL,
+	  { .ref.str_val = &cfg.trash_filefmt_zero },
+	},
+	{ "trash_filefmt_more", "", "file name when moved to the trash directory and another file exists with the same name",
```

Comment:
I think Vim has too many options, which makes it sometimes hard to find those you need. I'd have a single option with syntax akin to:
```vim
set trashfilefmt=zero:something,more:somethingelse
```
Like in `'fillchars'` option (`fillchars_handler()`).